### ISSUE TITLE
Include Istio version tested/supported in release notes

### ISSUE THREAD

**Issue by AngeloDanducci** [knative-sandbox/net-istio#958]
Had someone in Slack ask about supported Istio versions today but our documentation was out of date.  

After talking with Dave I think it makes sense to make the docs point directly to the releases page and then include what version of Istio we are supporting/testing in future releases.

**Comment by dprotaso** [comment]
Any chance we can update the last few releases to mention the version- ie. releases 1.3 to 1.6 are currently in our support window

**Comment by AngeloDanducci** [comment]
I can go back and update the old releases manually in the meantime 👍

**Comment by nak3** [comment]
+1

FYI net-gateway-api adds the release page by this script -https://github.com/knative-sandbox/net-gateway-api/blob/main/hack/release.sh#L69 .
Here is the result https://github.com/knative-sandbox/net-gateway-api/releases

**Comment by dprotaso** [comment]
Looks like we made the necessary changes to the website: https://knative.dev/docs/install/installing-istio/#supported-istio-versions

But the latest release doesn't mention the Istio version - https://github.com/knative-sandbox/net-istio/releases

Our release note automation [1][2] will scan PR bodies for `release-note` blocks and surface those. It still requires a release lead to update the release with these generated notes (hopefully this will be automated (https://github.com/knative/test-infra/issues/3655))

But if the person bumping Istio in a PR didn't include this message (or there was no bump) it's not going to show up.

We could do what @nak3 suggested and create a 'header' for the release note - that's probably the actionable thing for this issue

[1] https://github.com/kubernetes/release/tree/master/cmd/release-notes
[2] https://github.com/knative/actions/blob/main/.github/workflows/release-notes.yaml)

**Comment by dprotaso** [comment]
I just updated the 1.9 release note with `Tested with Istio 1.16.0`

**Comment by Nageshbansal** [comment]
Hello @ReToCode @dprotaso, I noticed this issue and would love to contribute. Is it still open for someone to work on?

**Comment by ReToCode** [comment]
👍  Sure, feel free to assign it to yourself and start working on a PR.

**Comment by Nageshbansal** [comment]
Sure. And, the current issue is to update the release by adding generated notes and including headers for istio version. Is that correct?

**Comment by ReToCode** [comment]
Yes: https://github.com/knative-sandbox/net-istio/issues/958#issuecomment-1431743096

> We could do what @nak3 suggested and create a 'header' for the release note - that's probably the actionable thing for this issue

**Comment by dprotaso** [comment]
@Nageshbansal are you still interested in working on this issue?

**Comment by Nageshbansal** [comment]
@dprotaso , yes, I'll make the PR soon. Thanks

**Comment by thesuperzapper** [comment]
@dprotaso @ReToCode This is a very important usability issue, right now there is no way to know which versions of Istio are "officially supported" by what version of Knative.

The only way to know for sure is to test yourself, and issues might not expose themselves until after you have deployed to production.

I would be grateful for a list of Istio versions that each release ran its integration tests on, as this would probably be sufficient.
